tests/integration/test_vulnerability_detector/test_feeds/data/test_cases/cases_msu_inventory.yaml

```diff
@@ -0,0 +1,8 @@
+- name: Check Patches with no CVE present
```

Comment:
The `name` field helps us when launching a test, we can choose a specific test case by passing the name to the `-k` parameter, however, if the name contains spaces it may not work correctly, it is recommended to use `_` and all lowercase.

Reply:
done in https://github.com/wazuh/wazuh-qa/pull/4009/commits/2c11167dfbd10f2e4f190449c19831b52cde646a